addon/index.d.ts

```diff
@@ -648,6 +659,73 @@ type Settlement<T> = { state: 'fulfilled', value: T } | { state: 'rejected', rea
 
 type Settled<T> = Settlement<Resolved<T>>;
 
+// Decorator option types from ember-concurrency-decorators
```

Comment:
I know the plan is eventually to bring e-c-d in, but in the meantime for consistency is there a reason not to re-export these types the same way you're re-exporting the runtime values?

Reply:
I'm not sure, but they aren't exported in e-c-d, so I didn't want to expose them until I have an understanding for why they were not exported there.